src/hooks/useRoom.ts

```diff
@@ -3,13 +3,12 @@ import { useCallback, useMemo } from "react";
 import { Room } from "types/rooms";
 
 import { enterExternalRoom } from "utils/userLocation";
-import { orderedVenuesSelector } from "utils/selectors";
 import { enterVenue } from "utils/url";
 import { getExternalRoomSlug } from "utils/room";
 
-import { useSelector } from "hooks/useSelector";
 import { useRecentLocationUsers } from "hooks/users";
 import { useUser } from "hooks/useUser";
+import { useRelatedVenues } from "./useRelatedVenues";
```

Comment:
Use canonical import path + alphabetical sorting:

```suggestion
import { useRelatedVenues } from "hooks/useRelatedVenues";
import { useRecentLocationUsers } from "hooks/users";
import { useUser } from "hooks/useUser";
```